applications.yaml

```
@@ -246,9 +246,15 @@
   - border_overflow
   - tray_and_multi_window
 - name: Everything
+  identifier:
+    kind: Class
+    id: EVERYTHING
+  options:
+  - tray_and_multi_window
+- name: Everything1.5a
   identifier:
     kind: Exe
-    id: Everything.exe
+    id: Everything64.exe
   options:
   - tray_and_multi_window
```

```suggestion
    kind: Class
    id: EVERYTHING_(1.5a)
  options:
  - force
  - tray_and_multi_window
```

It is preferred to use window classes over process names if possible.